lnrpc/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.15.6-buster
+
+RUN apt-get update && apt-get install -y \
+  git \
+  protobuf-compiler='3.6.1*' \
+  clang-format='1:7.0*'
+
+ARG PROTOC_GEN_VERSION
```

Comment:
Consider providing defaults for these args.

Reply:
I explicitly didn't want to add any default values to make sure they're always set. Added a comment to explain.